application/src/main/java/org/opentripplanner/standalone/server/DefaultServerRequestContext.java

```diff
@@ -254,6 +259,11 @@ public TriasApiParameters triasApiParameters() {
     return triasApiParameters;
   }
 
+  @Override
+  public GtfsAPIParameters gtfsAPIParameters() {
```

Comment:
We've standardised on the spelling of `*Api*` in Javacode. It's a bit weird but consistent.